mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/processing/building/impl/PojoIndexedEmbeddedIdentityMappingCollector.java

```diff
@@ -28,6 +29,7 @@
 
 	private BoundPojoModelPathPropertyNode<?, ?> identifierModelPath;
 	private IdentifierBinder identifierBinder;
+	private Map<String, Object> params;
```

Comment:
```suggestion
	private Map<String, Object> params = Collections.emptyMap();
```

Because `contributeIdentifierField` may end up using this default value, and we don't want a `null` map.